apps/emqx/src/emqx_persistent_session_ds/emqx_persistent_session_ds_router.erl

```diff
@@ -83,117 +93,104 @@ init_tables() ->
             ]}
         ]}
     ]),
+    %% NOTE
+    %% Holds non-wildcard routes with any scopes other than the `any` scope.
+    ok = mria:create_table(?PS_ROUTER_EXT_TAB, [
```

Comment:
Do we need new tables for backward compatibility?

Reply:
Yeah, sort of. I haven't found a good way to structure the existing table so that rolling upgrades are safe (suggestions are welcome).